templates/timesyncd.conf.j2

```diff
@@ -0,0 +1,6 @@
+# vi: ft=dosini:
```

Comment:
My role is complete, however there is the chance that the systemd developers change the interface or add new options. It is not just my local editor, it is the de-facto standard in the team everyone who is still active has agreed on, except for you.